schematic_db/synapse/synapse.py

```diff
@@ -304,6 +286,11 @@ def delete_table_rows(self, synapse_id: str, data: pd.DataFrame) -> None:
             )
         self.syn.delete(sc.Table(synapse_id, data))
 
+    @retry(
```

Comment:
does the `SynapseHTTPError` still get raised if it persists after 5 attempts? I had to add a `retry_error_callback` in addition to the other parameters as well

Reply:
No, at the moment a Retry error gets called. Does that reraise the original error?

Reply:
The retry error includes the original error in the callback